src/client/components/SampleComponent/SlideshowImageComponent/SlideshowImageComponent.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './SlideshowImageComponent.styles.css';
+import ArrayOfImages from '../../../assets/data/ArrayOfImages';
```

Comment:
ArrayOfImages doesn't say much. What images?

Reply:
It should be import avatarImages from '../../../assets/data/ArrayOfImages. I have changed it on my code. I will push the changes tomorrow.